HHS/OPRE-OPS
# feat: ✨ adds Services Components to approve agreement page

## What changed

feat: adds Services Components to approve agreement page

## Issue

#2349 #2097 

## How to test

1. create a budget/status change
2. Goto approve page
3. see Services components
4. 💰 profit

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
